graphix/channels.py

```diff
@@ -0,0 +1,208 @@
+    """
+    return KrausChannel(
+        [
+            {"coef": np.sqrt(1 - prob), "operator": np.eye(2)},
+            {"coef": np.sqrt(prob / 3.0), "operator": Ops.x},
+            {"coef": np.sqrt(prob / 3.0), "operator": Ops.y},
+            {"coef": np.sqrt(prob / 3.0), "operator": Ops.z},
+        ]
+    )
+
+
+def pauli_channel(px: float, py: float, pz: float) -> KrausChannel:
+    """single-qubit pauli channel
+    .. math::
+        (1-p) \rho + p_X X * \rho * X + p_Y * Y * rho * Y + p_Z * Z * rho * Z) 
+    but my format is better with X, Y Z
+    """
+    if px + py + pz > 1:
+        raise ValueError('The sum of probabilities must not exceed 1.')
+    pI = 1 - px - py - pz
+    return KrausChannel(
+        [
+            {"coef": np.sqrt(1 - pI), "operator": np.eye(2)},
+            {"coef": np.sqrt(px / 3.0), "operator": Ops.x},
+            {"coef": np.sqrt(py / 3.0), "operator": Ops.y},
+            {"coef": np.sqrt(pz / 3.0), "operator": Ops.z},
+        ]
+    )
+
+
+def two_qubit_depolarising_channel(prob: float) -> KrausChannel:
+    """two-qubit depolarising channel.
+    .. math::
+        \mathcal{E} (\rho) = (1-p) \rho + \frac{p}{15}  \sum_{P_i \in \{id, X, Y ,Z\}^{\otimes 2}/(id \otimes id)}P_i \rho P_i
+
+    Parameters
+    ----------
+    prob : float
+        The probability associated to the channel
+
+    Returns
+    -------
+    :class:`graphix.channel.KrausChannel` object
+        containing the corresponding Kraus operators
+    """
+
+    return KrausChannel(
+        [
+            {"coef": np.sqrt(1 - prob), "operator": np.kron(np.eye(2), np.eye(2))},
+            {"coef": np.sqrt(prob / 15.0), "operator": np.kron(Ops.x, Ops.x)},
+            {"coef": np.sqrt(prob / 15.0), "operator": np.kron(Ops.z, Ops.z)},
+            {"coef": np.sqrt(prob / 15.0), "operator": np.kron(Ops.x, np.eye(2))},
+            {"coef": np.sqrt(prob / 15.0), "operator": np.kron(Ops.y, Ops.y)},
+            {"coef": np.sqrt(prob / 15.0), "operator": np.kron(Ops.y, np.eye(2))},
+            {"coef": np.sqrt(prob / 15.0), "operator": np.kron(Ops.z, np.eye(2))},
+            {"coef": np.sqrt(prob / 15.0), "operator": np.kron(np.eye(2), Ops.x)},
+            {"coef": np.sqrt(prob / 15.0), "operator": np.kron(np.eye(2), Ops.y)},
+            {"coef": np.sqrt(prob / 15.0), "operator": np.kron(np.eye(2), Ops.z)},
+            {"coef": np.sqrt(prob / 15.0), "operator": np.kron(Ops.x, Ops.y)},
+            {"coef": np.sqrt(prob / 15.0), "operator": np.kron(Ops.x, Ops.z)},
+            {"coef": np.sqrt(prob / 15.0), "operator": np.kron(Ops.y, Ops.x)},
+            {"coef": np.sqrt(prob / 15.0), "operator": np.kron(Ops.y, Ops.z)},
+            {"coef": np.sqrt(prob / 15.0), "operator": np.kron(Ops.z, Ops.y)},
+            {"coef": np.sqrt(prob / 15.0), "operator": np.kron(Ops.z, Ops.x)},
+        ]
+    )
+
+
+def two_qubit_depolarising_tensor_channel(prob: float) -> KrausChannel:
```

Comment:
how about adding two qubit dephasing channel and pauli channel?

Reply:
Random Pauli channel added in [5501114](https://github.com/TeamGraphix/graphix/pull/64/commits/5501114f4c9135bb49fee29db37721836a9e71be). 
Isn't the 2-qubit dephasing channel just the tensor product of single-qubt dephasing channels? Indedd, some channel manipulation utilities could be added (tensor, composition).

Reply:
great, how does it work? shall we move it to `channels` instead, that seems useful?

Reply:
How does what work?

I have no opinion on whether to move the random Pauli channels to `channel` or leave it in `random_objects` to me it made sense to gather all random stuff at the same place. 

I planned to add all Channel manipulation stuff a bit later.

Reply:
> > How does what work?
> 
> `rand_Pauli_channel_kraus` function.
You select the number of operators you want in the Channel (rank) from all possible Pauli tensor products that are precomputed, you build a discrete probability distribution with this number of possibilities and then you choose randomly the correct number of operators.


> Let's move things in `random_objects.py` outside of `tests`, perhaps? functions in `random_circuit` is not so generalizable (beyond unit tests) but I feel that things in `random_objects.py` that you added could be quite useful.

Agreed. Random channels can also be useful.